data/json/recipes/other/vehicles.json

```
@@ -2080,5 +2080,73 @@
     "using": [ [ "welding_standard", 25 ], [ "steel_standard", 5 ] ],
     "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "HAMMER", "level": 2 }, { "id": "SAW_M", "level": 2 } ],
     "components": [ [ [ "sheet_metal", 2 ], [ "sheet_metal_small", 48 ] ], [ [ "pipe", 6 ] ], [ [ "chain", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "wood_propeller_item",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_VEHICLE",
+    "skill_used": "fabrication",
+    "skills_required": [ "mechanics", 4 ],
+    "difficulty": 5,
+    "time": "2 h",
+    "reversible": true,
+    "decomp_learn": 5,
+    "book_learn": [
+      [ "textbook_fabrication", 6 ],
+      [ "welding_book", 6 ],
+      [ "book_icef", 4 ],
+      [ "textbook_mechanics", 4 ],
+      [ "manual_mechanics", 4 ],
+      [ "reference_fabrication1", 6 ]
+    ],
+    "qualities": [
+      { "id": "SCREW", "level": 1 },
+      { "id": "HAMMER", "level": 2 },
+      { "id": "SAW_W", "level": 2 },
+      { "id": "CUT", "level": 1 }
+    ],
+    "components": [ [ [ "log", 1 ] ], [ [ "2x4", 2 ] ], [ [ "nail_glue", 20, "LIST" ] ], [ [ "chain", 1 ] ], [ [ "scrap", 10 ] ] ]
```

This rigs it so 2 heavy sticks or 1 long stick can be used too.
```suggestion
    "components": [ [ [ "log", 1 ] ], [ [ "wood_structural", 1, "LIST" ] ], [ [ "nail_glue", 20, "LIST" ] ], [ [ "chain", 1 ] ], [ [ "scrap", 10 ] ] ]
```
Side note, a whole log is perhaps a bit much as an alternative...